app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
```

Comment:
- Hi, it's a good practice to define a new method on `comment`, by creating a new comment, the new method can make a @comment instance variable accessible in the view:
```ruby
 def new
    @comment = Comment.new
  end
```
I would suggest you add this code above

Reply:
@OLIPLICHE Thanks for your valuable suggestion. I just updated the required change. Please review my work. Thank you.